back/src/queue/jobs/administrativeTransfer.ts

```diff
@@ -48,5 +49,34 @@ export async function processAdministrativeTransferJob(
     }
   );
 
+  // BSDAs
+  const bsdasToTransfer = await prisma.bsda.findMany({
+    where: {
+      destinationCompanySiret: fromOrgId,
+      status: {
+        in: ["AWAITING_CHILD"]
+      }
+    },
+    select: { id: true }
+  });
+
+  const bsdaRepository = getBsdaRepository({
+    auth: AuthType.Bearer,
+    id: "JOB_ADMINISTRATIVE_TRANSFER",
+    name: "JOB_ADMINISTRATIVE_TRANSFER"
+  } as Express.User);
+
+  await bsdaRepository.updateMany(
+    { id: { in: bsdasToTransfer.map(bsda => bsda.id) } },
+    {
+      destinationCompanySiret: toCompany.orgId,
+      destinationCompanyName: toCompany.name,
+      destinationCompanyAddress: toCompany.address,
+      destinationCompanyContact: toCompany.contact,
+      destinationCompanyMail: toCompany.contactEmail,
+      destinationCompanyPhone: toCompany.contactPhone
+    }
+  );
```

Comment:
Pourquoi récupérer les ids pour ensuite updateMany?
Tu pourrais juste faire le updateMany avec comme query ce qu'il y a dans le findMany non? Ca évite de charger la mémoire pour rien et c'est plus opti

Reply:
et remplacer `in: ["AWAITING_CHILD"]` par status: "AWAITING_CHILD" a priori

Reply:
Ah et je vois que @Riron a fait ça aussi pour le BSDD, donc si j'ai pas loupé un truc et que ça marche aussi pour BSDD je veux bien que tu fasse pareil histoire d'être cohérent

Reply:
ça marche pas pour les BSDD parce que la fonction `updateMany` du repository ne prend pas un `where`, mais une liste d'ID. C'est pour ça qu'il a fait ça comme ça. Et moi j'ai copié-collé comme un singe.

J'ai fait tes modifs!